ContestUtil/src/org/icpc/tools/contest/util/cms/JsonToTSVConverter.java

```diff
@@ -184,7 +184,7 @@ public boolean accept(File f) {
 			groupList.sort((g1, g2) -> compare(g1.getId(), g2.getId()));
 			teamList.sort((t1, t2) -> compare(t1.getId(), t2.getId()));
 			orgList.sort((i1, i2) -> compare(i1.getId(), i2.getId()));
-			personList.sort((m1, m2) -> compare(m1.getTeamId(), m2.getTeamId()));
+			personList.sort((m1, m2) -> compare(m1.getId(), m2.getId()));
```

Comment:
We could sort *first* by teamID and then by person ID if we still want teams together.

Reply:
Thought about that, but wasn't sure it was worth the code :). Will see how this goes, can always switch to that.